src/modules/user/components/user-services/user-services.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+
+// components
+import { Typography } from '@/modules/core/components';
+import { ServiceTag, ServiceEntity } from '@/modules/service/components';
+
+import styles from './user-services.module.scss';
+
+export const Services = () => {
+  return (
+    <div className={styles.container}>
+      <Typography className={styles.title} variant='title'>
+        Services
+      </Typography>
+      <div className={styles.service}>
+        <ServiceTag service='hair' />
+        <ServiceEntity
+          serviceName='Haircut for children'
+          serviceTime='40min'
+          servicePrice='25 $'
+        />
+        <ServiceEntity
+          serviceName='Haircut for adults'
+          serviceTime='1h'
+          servicePrice='25 $'
+        />
+        <ServiceEntity
+          serviceName='Haircut for adults + beard trimming'
+          serviceTime='1h 30min'
+          servicePrice='25 $'
+        />
+      </div>
```

Comment:
Remove component `ServiceEntity` and create `ServicesTable` instead.

Example:

```tsx
import type { SupportedService } from '@/modules/service/types/service.types';

export type ServicesTableProps = {
  service: SupportedService;
  userServices: {
    name: string;
    duration: string;
    price: string;
  }[]
  
}

export const ServicesTable: FC<ServicesTableProps> = ({ service, userServices }) => {
  return (
    <div className={styles.root}> // styles.services now
      <ServiceTag service={service} />
      {userServices.map((userService) => (
         <div>{userService.name} ... <div/>
      ))}
    </div>
  )
}
```

Then, use it next way:

```tsx
<Typography className={styles.title} variant='title'>
  Services
</Typography>
<ServicesTable
  service='hair'
  userServices={[{
    name: 'Haircut for children',
    duration: '40min',
    price: '25 $'
  }, {
    name: ...
  }]}
 />
```